### DISCUSSION TITLE
EssentialsX-Discord: send "/tellraw @a" (and similar) to chat channel.

### DISCUSSION THREAD

**Discussion by EvModder** [EssentialsX/Essentials#5123] [Ideas and Feature Suggestions] [answered]
A config option to do this would be nice, since `/tellraw @a` and similar vanilla broadcast-style commands are relatively common.

I thought perhaps enabling `show-all-chat: true` would do this, but it seems not

**Comment by JRoy** [comment, marked as answer]
There is no event for tellraw, would require a Paper/Bukkit API addition and I don't see many people using this.
